model/client.yaml

```diff
@@ -12,27 +12,35 @@ groups:
     attributes:
       - id: address
         type: string
-        brief: Client address - unix domain socket name, IPv4 or IPv6 address.
+        brief: Client address - IP address or Unix domain socket name.
         note: >
           When observed from the server side, and when communicating through an intermediary, `client.address` SHOULD represent
-          client address behind any intermediaries (e.g. proxies) if it's available.
+          the client address behind any intermediaries (e.g. proxies) if it's available.
         examples: ['/tmp/my.sock', '10.1.2.80']
       - id: port
```

Comment:
(Not related to symmetry, but it seems that the network.transport attribute was moved to trace-specific attributes, and the port is ambiguous if we don't know whether it was UDP or TCP)

Reply:
can you create an issue for this?

Reply:
created #250